2_Lists/interquartile_range/readme.md

```diff
@@ -66,6 +66,22 @@ class Node {
 };
 ```
 
+### Restriction:
+For this problem, we ask that you use properties of lists rather than using another container,
+for example you cannot move all of the elements into a vector and then find the IQR of the vector.
+Additionally, you should not count the elements of the list to use e.g. count/4 and 3*count/4
```

Comment:
**Reframe this to:** 
Additionally, you should not count the elements of the list and use this count to determine the locations of Q1 and Q3 in the list (e.g. count/4 and 3*count/4).